docs/src/main/asciidoc/config-mappings.adoc

```diff
@@ -417,6 +417,40 @@ server.form.landing-page=index.html
 The configuration property needs to specify an additional name to act as the key. In this case the `form()` `Map` will
 contain three elements with the keys `login-page`, `error-page` and `landing-page`.
 
+Mapping can also be done as a Map of objects.
+[source,java]
+----
+@ConfigMapping(prefix = "server")
+public interface Servers {
+    @WithParentName
+    Map<String, Server> allServers();
+}
+
+@ConfigGroup
```

Comment:
The `@ConfigGroup` annotation is not required.

Reply:
Done. please let me know if you need me to make any other change.